src/net/udp.rs

```diff
@@ -92,6 +92,62 @@ impl UdpSocket {
         Ok(UdpSocket { inner: socket })
     }
 
+    /// Creates new `UdpSocket` from a previously bound `std::net::UdpSocket`.
+    ///
+    /// This function is intended to be used to wrap a UDP socket from the
+    /// standard library in the Tokio equivalent. The conversion assumes nothing
+    /// about the underlying socket; it is left up to the user to set it in
+    /// non-blocking mode.
```

Comment:
Nonblocking isn't as meaningful in the context of io-uring.

Reply:
Took a stab at an updated description that isn't as specific - plesae let me know what you think.

Reply:
So, the requirement to set things to nonblocking mode is more relevant in the context of epoll, where the strategy is to basically loop on attempting a non-blocking read/write up front, and, if that fails, registering with the IO driver's epoll loop to be woken once readiness is signaled, and to then go back to attempting an IO op again, which mail once again fail, and so on...